ui/app/templates/components/diff-version-selector.hbs

```diff
@@ -0,0 +1,94 @@
+<Toolbar>
+  <div class="version-diff-toolbar" data-test-version-diff-toolbar>
+    {{!-- Left side version --}}
+    <BasicDropdown
+      @class="popup-menu"
+      @horizontalPosition="auto-right"
+      @verticalPosition="below"
+      as |D|
+    >
+      <D.trigger
+        data-test-popup-menu-trigger="left-version"
+        @class={{concat "toolbar-link" (if D.isOpen " is-active")}}
+        @tagName="button"
+      >
+        Version {{or this.leftSideVersionSelected @model.currentVersion}}
+        <Chevron @direction="down" @isButton={{true}} />
+      </D.trigger>
+      <D.content @class="popup-menu-content">
+        <nav class="box menu">
+          <ul class="menu-list">
+            {{#each (reverse @model.versions) as |leftSideSecretVersion|}}
+              <li class="action" data-test-leftSide-version={{leftSideSecretVersion.version}}>
+                <button 
+                  class="link"
+                  {{on "click" (fn this.selectVersion leftSideSecretVersion.version D.actions "left")}}
+                  >
+                  Version {{leftSideSecretVersion.version}}
+                  {{#if (and (eq leftSideSecretVersion.version (or this.leftSideVersionSelected this.model.currentVersion)) (not leftSideSecretVersion.destroyed) (not leftSideSecretVersion.deleted))}}
```

Comment:
It looks like `.destroyed` and `.deleted` exist on the `metadata` not `data` so we don't have access to those attributes here. I don't these icons are displaying?

Reply:
This was resolved. Discovered it was an issue in main an is being addressed.